profiling/roctx-connector/CMakeLists.txt

```diff
@@ -1,7 +1,18 @@
-find_library(ROCM_ROCTX_LIB roctx64 REQUIRED HINTS $ENV{ROCM_PATH}/lib)
-find_path(ROCM_ROCTX_INCLUDE roctx.h REQUIRED HINTS $ENV{ROCM_PATH}/include/roctracer)
-
 kp_add_library(kp_roctx_connector kp_roctx_connector.cpp)
 
-target_include_directories(kp_roctx_connector PRIVATE ${ROCM_ROCTX_INCLUDE})
-target_link_libraries(kp_roctx_connector PRIVATE ${ROCM_ROCTX_LIB})
+# As of ROCm 6.2, it is recommended to use ROCtx provided by rocprofiler-sdk-roctx
+# instead of the "old" one provided by roctracer.
+#
+# See also: https://rocm.docs.amd.com/projects/rocprofiler-sdk/en/amd-mainline/how-to/using-rocprofv3.html
+find_package(rocprofiler-sdk-roctx CONFIG PATHS $ENV{ROCM_PATH} PATH_SUFFIXES "lib/cmake/rocprofiler-sdk-roctx")
```

Comment:
Why are we specifying `PATH_SUFFIXES`?  I expect that path will be constructed by CMake regardless.

Reply:
Also, why do you think we should add `ROCM_PATH` to the search paths?

Reply:
OK. The `PATH_SUFFIXES` shouldn't be needed indeed. I still have to check for the `ROCM_PATH`.

Reply:
I've just tested the need for the `ROCM_PATH`. If we don't specify it as a search path, then the package isn't found.